dynamo/tools/clustering.py

```diff
@@ -195,15 +196,20 @@ def leiden(
 
     Parameters
     ----------
+    adata
+        The annotated data matrix.
+    resolution : float, optional
```

Comment:
remove space between argument name and ":"
remove float, optional, because this is not in google code style